.github/workflows/deploy_preview_with_reports.yml

```diff
@@ -0,0 +1,238 @@
+name: Deploy Preview (with VRT/AAT reports)
+on:
+  workflow_run:
+    workflows: [CI]
+    types:
+      - completed
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    if: ${{ always() && github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.head_repository.full_name == 'primer/react' }}
```

Comment:
Added `github.event.workflow_run.event == 'pull_request'` checks to each job to only run for `pull_request` events